### PR TITLE
TA-3004 added success stories and unit tests

### PR DESCRIPTION
**Pull request by averysmithproductions**
for jira ticket https://us-sba.atlassian.net/browse/TA-3004
- [x] featured images are displayed on blog cards
- [x] Ensure the addition of images on the page links to blog detail page
- [x] show a case when there is no image available

### CODE REVIEW

File: src/client/components/pages/blogs-landing/blogs-landing.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import { fetchRestContent, fetchSiteContent } from '../../../fetch-content-helper'
-import { AuthorCardCollection, BlogCategoryDeck, Hero } from 'organisms'
+import { AuthorCardCollection, BlogCategoryDeck, Hero, SuccessStories } from 'organisms'
```
Comment (amyd11) [inline src/client/components/pages/blogs-landing/blogs-landing.jsx]: Looks like `SuccessStories` is an unused import

File: src/client/components/organisms/blog-category-deck/blog-category-deck.jsx [hunk 2]
```diff
@@ -7,16 +7,24 @@ import { CardCollection } from 'organisms'
 import styles from './blog-category-deck.scss'
 
 class BlogCategoryDeck extends React.Component {
-  reformatCard(card) {
-    return {
-      italicText: moment.unix(card.created).format('MMMM D, YYYY'),
+  reformatCard({created, url, summary, title, blogBody}) {
```
Comment (amyd11) [inline src/client/components/organisms/blog-category-deck/blog-category-deck.jsx]: Is this destructuring `card` as it's coming into the function? Cool, I didn't know you could do that

File: src/client/components/pages/blogs-landing/blogs-landing.jsx [hunk 3]
```diff
@@ -34,6 +34,13 @@ class BlogsLandingPage extends Component {
         subtitle: 'Commentary and advice from leaders in the small business industry.',
         url: '/blogs/industry-word',
         cards: []
+      },
+      {
+        category: 'Success Story',
+        title: 'Success Stories posts',
```
Comment (amyd11) [inline src/client/components/pages/blogs-landing/blogs-landing.jsx]: I think the title needs to be `Success Story posts` instead

File: src/client/components/organisms/blog-category-deck/blog-category-deck.jsx (hunk 2 above)
Comment (averysmithproductions) [reply src/client/components/organisms/blog-category-deck/blog-category-deck.jsx]: yes, it is.

**Pull request merged**